howtos/how-to-make-a-threaded-chat/server.js

```diff
@@ -0,0 +1,228 @@
+import express from "express";
+import cors from "cors";
+import fetch from "node-fetch";
+
+const appId = "<APP_ID>";
+const secretKey = "<SECRET_KEY>";
+
+const basePath = "https://api.talkjs.com";
+
+const app = express();
+app.use(cors());
+app.use(express.json());
+
+app.listen(3000, () => console.log("Server is up"));
+
+const senderId = `threadsExampleSender`;
+const receiverId = `threadsExampleReceiver`;
+
+// Create a thread as a new conversation
+async function createThread(parentMessageId, parentConvId, participants) {
+  const conversationId = "replyto_" + parentMessageId;
+  return fetch(`${basePath}/v1/${appId}/conversations/${conversationId}`, {
+    method: "PUT",
+    headers: {
+      "Content-Type": "application/json",
+      Authorization: `Bearer ${secretKey}`,
+    },
+    body: JSON.stringify({
+      participants: participants,
+      subject: "Replies",
+      custom: {
+        parentConvId: parentConvId,
+        parentMessageId: parentMessageId,
+      },
+    }),
+  });
+}
+
+async function duplicateParentMessageText(parentMessageId, messageText) {
+  const conversationId = "replyto_" + parentMessageId;
+  return fetch(
+    `${basePath}/v1/${appId}/conversations/${conversationId}/messages`,
+    {
+      method: "POST",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${secretKey}`,
+      },
+      body: JSON.stringify([
+        {
+          text: messageText,
+          type: "SystemMessage",
+        },
+      ]),
+    }
+  );
+}
+
+async function updateReplyCount(messageId, conversationId, count) {
+  return fetch(
+    `${basePath}/v1/${appId}/conversations/${conversationId}/messages/${messageId}`,
+    {
+      method: "PUT",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${secretKey}`,
+      },
+      body: JSON.stringify({
+        custom: { replyCount: count.toString() },
+      }),
+    }
+  );
+}
+
+function getMessages(messageId) {
+  const conversationId = "replyto_" + messageId;
+
+  return fetch(
+    `${basePath}/v1/${appId}/conversations/${conversationId}/messages`,
+    {
+      method: "GET",
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${secretKey}`,
+      },
+    }
+  );
+}
+
+app.post("/newThread", async (req, res) => {
+  // Get details of the message we'll reply to
+  const parentMessageId = req.body["messageId"];
+  const parentConvId = req.body["conversationId"];
+  const parentMessageText = req.body["messageText"];
+  const parentParticipants = req.body["participants"];
+
+  let response = await getMessages(parentMessageId);
```

Comment:
No need for both the message and the response to be a let

Reply:
Updated to `const`